packages/help-center/src/components/help-center-contact-form.tsx

```diff
@@ -441,7 +441,7 @@ export const HelpCenterContactForm = () => {
 					<Button
 						disabled={ gptFetching ? true : isCTADisabled() }
 						onClick={ handleCTA }
-						primary
+						primary={ true }
```

Comment:
Was trying to figure out an earlies version of the following error from the unit tests:
```
packages/help-center/src/components/help-center-contact-form.tsx(444,7): error TS2769: No overload matches this call.
    Overload 1 of 2, '(props: Props, context?: any): ReactElement<any, any> | Component<Props, any, any> | null', gave the following error.
      Type '{ children: string | undefined; disabled: boolean; onClick: () => void; primary: boolean; className: string; }' is not assignable to type 'IntrinsicAttributes & Props'.
        Property 'primary' does not exist on type 'IntrinsicAttributes & Props'. Did you mean 'isPrimary'?
```
But My IDE tells me the opposite.

<img width="902" alt="Screenshot 2023-04-24 at 14 30 50" src="https://user-images.githubusercontent.com/844866/233984037-9dff1689-02ce-482c-8005-b8e9298352f5.png">